src/DaPigGuy/PiggyCustomEnchants/enchants/tools/hoe/FarmerEnchant.php

```diff
@@ -51,7 +51,7 @@ public function react(Player $player, Item $item, Inventory $inventory, int $slo
             if ($block instanceof Crops) {
                 $seed = $block->getPickedItem();
                 if ($player->getInventory()->contains($seed)) {
-                    CustomEnchantManager::getPlugin()->getScheduler()->scheduleDelayedTask(new ClosureTask(function (int $currentTick) use($player, $seed, $block) : void {
+                    CustomEnchantManager::getPlugin()->getScheduler()->scheduleDelayedTask(new ClosureTask(function () use($player, $seed, $block) : void {
```

Comment:
There shouldn't be a space before the colon (``use($player, $seed, $block) : void {`` -> ``use($player, $seed, $block): void {)``)